src/docs/teams/account/redeeming-a-coupon.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Teams | Redeeming a coupon
+layout: docs.hbs
+---
+
+# Redeeming a coupon
+
+Redeeming a coupon for a Truffle Teams paid subscription is different for new and existing subscriptions. Please be sure to read the instructions carefully to make sure your coupon is applied correctly. If at any point you need some help, please don't hesitate to [ask us a question](/docs/teams/support/ask-us-a-question).
+
+- [New Subscriptions](#new-subscriptions)
+- [Existing Subscriptions](#existing-subscriptions)
+
+## New Subscriptions
+
+Before you can continue below, you'll need to make sure you have [created an account](/docs/teams/getting-started/creating-an-account) **and** [added at least one repository](/docs/teams/getting-started/adding-repositories) for the GitHub user/organization you'd like to apply your coupon code for.
+
+Once you've made sure the account you'd like to apply the coupon for exists, navigate to the <a href="https://my.truffleteams.com/#/settings" target="_blank">settings</a> page. You can also navigate there by clicking on your avatar/username in the left sidebar.
+
+Next, click on the GitHub user or organization in the sidebar that you'd like to redeem the coupon for.
+
+![Account Select](/img/docs/teams/coupon-redemption-account-selection.png)
+
+Verify you are on the correct account by looking at the top bar that says **Settings for ...**
+
+![Account Verification](/img/docs/teams/coupon-redemption-account-verification.png)
+
+Next, verify you're on the **BILLING** tab.
+
+![Billing Tab](/img/docs/teams/coupon-redemption-billing-tab.png)
+
+You should see that **FREE** is your current plan like the below picture; if it is not, please follow the instructions [below](#existing-subscriptions) for redeeming the coupon for existing subscriptions.
+
+![Verification of Free Plan](/img/docs/teams/coupon-redemption-free-verification.png)
+
+Press **<span class="inline-button">UPGRADE</span>** under the **PRO** plan.
+
+Confirm/update the **Billing Email** so that it is correct. This is the email that will receive receipts and failed payment notices.
+
+![Billing Email](/img/docs/teams/coupon-redemption-billing-email.png)
+
+Press **<span class="inline-button">CONTINUE</span>**.
+
+Enter your location information and press **<span class="inline-button">Continue</span>**.
```

Comment:
```suggestion
Enter your location information and press **<span class="inline-button text-light">Continue raquo;</span>**.
```
Just bringing the formatting exactly in line with Paddle.

Reply:
I know the `<u></u>` element is especially stinky, but it's recommended by Boostrap and works as a quick-n-dirty update (same applies to similar edit below).

Reply:
I'm not quite sure what you're referring to here with `<u></u>` since it's not in your suggestion?

Further, I'm applying your suggestion in a separate commit as you forgot the `&` in `&raquo;`